llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -22,11 +22,17 @@
 //             uses the record name by replacing Feature with Has.
 // value     - Value to assign to the field in RISCVSubtarget when this
 //             extension is enabled. Usually "true", but can be changed.
-class RISCVExtension<string name, int major, int minor, string desc,
+class RISCVExtension<int major, int minor, string desc,
                      list<SubtargetFeature> implies = [],
                      string fieldname = !subst("Feature", "Has", NAME),
-                     string value = "true">
-    : SubtargetFeature<name, fieldname, value, desc, implies> {
+                     string value = "true", string prefix = "">
```

Comment:
Can we use the `IsExperimental` field to create the "experimental-" prefix?

Reply:
Done. Thanks!